src/main/java/it/pagopa/pn/paperchannel/utils/PaperCalculatorUtils.java

```diff
@@ -84,7 +84,7 @@ public Mono<ShipmentCalculateResponse> costSimulator(String tenderId, ShipmentCa
     private BigDecimal getCostSimulated(PnPaperChannelCostDTO contract, Integer numSides, Integer pageWeight, String productType, boolean isReversePrinter) {
         Integer numbersPage = isReversePrinter ? (int) Math.ceil(((double) numSides)/2) : numSides;
         Integer finalPageWeight = pageWeight == null ? pnPaperChannelConfig.getPaperWeight() : pageWeight;
-        Integer totalPagesWeight = numbersPage * finalPageWeight;
+        Integer totalPagesWeight = (numbersPage * finalPageWeight) + pnPaperChannelConfig.getLetterWeight();
         return getSimplifiedAmount(totalPagesWeight, numbersPage, numbersPage, contract, productType);
```

Comment:
E' giusto che getSimplifiedAmount sia invocata con `numbersPage` anche per `totPagesIgnoringAAR`?

Reply:
si è corretto perchè lato api non hai evidenza in merito alla presenza di eventuali AAR